composer.json

```diff
@@ -29,10 +29,11 @@
         "ext-json": "*",
         "guzzlehttp/guzzle": "^7.2",
         "illuminate/macroable": "^8.0|^9.0",
+        "laminas/laminas-stdlib": "^3.10",
```

Comment:
![awkward-in-trouble](https://user-images.githubusercontent.com/4172890/179832093-16f23195-56d5-4598-a137-ce52f60a1d5f.gif)

Maybe is a bit overkill importing the whole laminas framework std for just a wordwrapping?

Reply:
No, it is the only code that works with multibyte strings